tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -317,18 +317,21 @@ def run_conv_with_split(
     assert_with_pcc(torch_output_tensor, torch_out_golden_tensor, pcc=pcc)
 
 
+@skip_for_grayskull()
```

Comment:
Do other tests cover these test scenarios on grayskull?
If not, why would these test scenarios note be relevant for grayskull?

Reply:
We are not planning on supporting these new features on gray skull.

Reply:
Are the cases in the already existing test (before this PR) supported by grayskull?

Reply:
Most of them were skipped, due to an OOM error.  The width sharded Conv2d which is tested by this code was implemented to support a wormhole only model.